test/async-hooks/test-httpparser.response.js

```diff
@@ -19,11 +19,8 @@ const hooks = initHooks();
 hooks.enable();
 
 const request = Buffer.from(
-  'HTTP/1.1 200 OK' + CRLF +
-  'Content-types: text/plain' + CRLF +
-  'Content-Length: 4' + CRLF +
-  CRLF +
-  'pong'
+  `HTTP/1.1 200 OK${CRLF}Content-types: text/plain${CRLF}` +
+  `Content-Length: 4${CRLF}${CRLF}pong`
```

Comment:
@gibfahn I thought it was checking the misnamed header (It's `Content-types` here and not `Content-Type`) I assumed it was checking compatibility with nonconforming clients.